pydantic/main.py

```diff
@@ -345,8 +358,20 @@ def json(
 
         `encoder` is an optional function to supply as `default` to json.dumps(), other arguments as per `json.dumps()`.
         """
+        if skip_defaults is not None:
+            warnings.warn(
+                f'{type(self).__name__}.dict: "skip_defaults" is deprecated and replaced by "exclude_unset"',
```

Comment:
Does this warning look good to you?

Reply:
```suggestion
                f'{self.__class__.__name__}.dict(): "skip_defaults" is deprecated and replaced by "exclude_unset"',
```
I would do this, but otherwise looks fine.